vars/isHostReady.groovy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
+Boolean call(String gceInstance, String zone = 'us-central1-b') {
+  // Host is ready if GCE instance status is RUNNING
+  sh (
+    script: """
+      gcloud compute instances describe ${instance} --zone ${zone} | sed -n -e 's/^status: //p'
```

Comment:
you can use the format option to retrieve values directly:

`gcloud compute instances describe ${instance} --zone ${zone} --format='get(status)'`